File: server/src/main/java/ru/maximus/tictactoe/server/ServerLauncher.java
package ru.maximus.tictactoe.server;

import com.badlogic.gdx.backends.headless.HeadlessApplication;
import com.badlogic.gdx.backends.headless.HeadlessApplicationConfiguration;

/** Launches the server application. */
public class ServerLauncher {
    public static void main(String[] args) {
        HeadlessApplicationConfiguration config = new HeadlessApplicationConfiguration();

        new HeadlessApplication(new Server());
    }
}

File: server/src/main/java/ru/maximus/tictactoe/server/ServerLauncher.kt
package ru.maximus.tictactoe.server

import com.badlogic.gdx.backends.headless.HeadlessApplication
import com.badlogic.gdx.backends.headless.HeadlessApplicationConfiguration

/** Launches the server application.  */
fun main(args: Array<String>) {
    val config = HeadlessApplicationConfiguration()

    HeadlessApplication(Server())
}